modules/ROOT/pages/update-acm.adoc

```diff
@@ -21,6 +21,11 @@ The staging environment will have a partial or full copy of the data in the prod
 
 To update your API community:
 
+. Ensure your Salesforce organizations have the "External Data Change Tracking for OData 4.0 Adapter" feature enabled.
+.. Before upgrading the package in your production or sandbox organization, check that your organization has this feature enabled by going to *Setup* > *External Data Source* and editing the Exchange data source.
```

Comment:
production or sandbox environment? rather than organization?

Reply:
In this case org is correct. Each env has an org, and packages are upgraded in orgs.